docs/topics/wildfly-swarm/docs/reference/fractions/elytron.adoc

```diff
@@ -0,0 +1,996 @@
+
+swarm.elytron.ldap-key-stores._KEY_.state:: 
+The state of the underlying service that represents this KeyStore at runtime, if it is anything other than UP runtime operations will not be available.
+
+swarm.elytron.ldap-realms._KEY_.allow-blank-password:: 
+Does this realm support blank password direct verification? Blank password attempt will be rejected otherwise.
+
+swarm.elytron.ldap-realms._KEY_.dir-context:: 
+The configuration to connect to a LDAP server.
+
+swarm.elytron.ldap-realms._KEY_.direct-verification:: 
+Does this realm support verification of credentials by directly connecting to LDAP as the account being authenticated?
+
+swarm.elytron.ldap-realms._KEY_.identity-mapping:: 
+The configuration options that define how principals are mapped to their corresponding entries in the underlying LDAP server.
+
+swarm.elytron.logical-permission-mappers._KEY_.left:: 
+Reference to the permission mapper to use to the left of the operation.
+
+swarm.elytron.logical-permission-mappers._KEY_.logical-operation:: 
+The logical operation to use to combine the permission mappers.
+
+swarm.elytron.logical-permission-mappers._KEY_.right:: 
+Reference to the permission mapper to use to the right of the operation.
+
+swarm.elytron.logical-role-mappers._KEY_.left:: 
+Reference to a role mapper to be used on the left side of the operation.
+
+swarm.elytron.logical-role-mappers._KEY_.logical-operation:: 
+The logical operation to be performed on the role mapper mappings.
+
+swarm.elytron.logical-role-mappers._KEY_.right:: 
+Reference to a role mapper to be used on the right side of the operation.
+
+swarm.elytron.mapped-regex-realm-mappers._KEY_.delegate-realm-mapper:: 
+The RealmMapper to delegate to if the pattern does not match. If no delegate is specified then the default realm on the domain will be used instead. If the username does not match the pattern and a delegate realm-mapper is present, the result of delegate-realm-mapper is mapped via the realm-map.
+
+swarm.elytron.mapped-regex-realm-mappers._KEY_.pattern:: 
+The regular expression which must contain at least one capture group to extract the realm from the name. If the regular expression matches more than one capture group, the first capture group is used.
+
+swarm.elytron.mapped-regex-realm-mappers._KEY_.realm-map:: 
+Mapping of realm name extracted using the regular expression to a defined realm name. If the value for the mapping is not in the map or the realm whose name is the result of the mapping does not exist in the given security domain, the default realm is used.
+
+swarm.elytron.mechanism-provider-filtering-sasl-server-factories._KEY_.available-mechanisms:: 
+The SASL mechanisms available from this factory after all filtering has been applied.
+
+swarm.elytron.mechanism-provider-filtering-sasl-server-factories._KEY_.enabling:: 
+When set to 'true' no provider loaded mechanisms are enabled unless matched by one of the filters, setting to 'false' has the inverse effect.
```

Comment:
same as above

Reply:
Note to self: not easily fixable in our sources.